starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +53,15 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setId("123456");
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    InventoryDAO invDao = new InventoryDAO(this.mongoTemplate);
```

Comment:
You don't need to initialize the inventory dao because the @setup method already does it for you